packages/compiler/src/plugins.ts

```diff
@@ -77,6 +78,27 @@ export const processMyst = () => {
           };
           (parent as Parent).children[index] = newNode;
           return index;
+        } else if (mystType === "panels") {
+          let cards = parsePanels(value);
+          console.log(cards);
+          // parse each card
+          const newNode = {
+            type: "html",
+            value: mustache.render(
+              `<Panels>
+               <div>
+               {{#cards}}
+               {{header}}
+               {{body}}
+               {{footer}}
+               {{/cards}}
+               </div>
+               </Panels>`,
+              cards
```

Comment:
It might make more sense to call micromark on the output of parsePanels here -- might be a slightly better seperation of concerns.

Reply:
micromark now called on output of ```parsePanel``` within ```plugins.ts```